src/pal/src/file/file.cpp

```diff
@@ -2505,6 +2482,7 @@ CorUnix::InternalSetEndOfFile(
     // extend the file to consume the remainder of free space.
     // 
     struct statfs sFileSystemStats;
+    off_t cbFreeSpace;
```

Comment:
@janvorli clang required it.  The decl created a protected scope, and the goto violated that.